### DISCUSSION TITLE
Можно гайд для глупеньких, как правильно запустить zapret на Android? Root (Magisk) есть.

### DISCUSSION THREAD

**Discussion by ImMALWARE** [bol-van/zapret#734] [Q&A] [answered]
Можно гайд для глупеньких, как правильно запустить zapret на Android? Root (Magisk) есть. 
Шаги те же, что и для Linux? Как правильно создавать daemon с zapret?

**Comment by bol-van** [comment]
Как в linux только iptables и команда запуска nfqws.
Все остальное не как в linux

**Reply by ImMALWARE** [reply]
Планируется ли гайд для глупеньких?)

**Reply by bol-van** [reply]
У меня традиция. все для глупеньких отдавать на откуп в дискуссии и на форумы.
Глупенькие для глупеньких лучше обьяснят

**Comment by Fenrir-0xFF** [comment, marked as answer]
Скрипт для андроида https://github.com/bol-van/zapret/issues/714

**Comment by TvoyOtchim650** [comment]
И где гайд??

**Reply by HentayHentay** [reply]
Где-то в дискуссиях. 
Гайд - поиск и чтение и понимание, что кирпич можно родить
